ko/admin/control.rst

```diff
@@ -152,19 +152,8 @@ CUBRID 자바 저장 프로시저 (Java SP) 서버 프로세스를 제어하기
 *   restart: 자바 저장 프로시저 서버 프로세스 재시작
 *   status: 자바 저장 프로시저 서버 프로세스 상태 확인
 
-모든 명령어는 특정 데이터베이스 이름 (**[database_name]**) 을 인수로 지정할 수 있으며, 데이터베이스 이름을 지정하지 않으면 cubrid.conf의 **[service]** 섹션의 **server** 프로퍼티에서 데이터베이스 이름을 참조한다.
-
-::
-
-    # cubrid.conf
-
-    [service]
-
-    ...
-
-    server=demodb,testdb
-
-    ...
+| 모든 명령어는 특정 데이터베이스 이름 (**[database_name]**) 을 인수로 지정할 수 있다.
```

Comment:
Did you intentionally subtract the default value when databasename is not specified?

Reply:
I'm gonna fix the example not to omit the database name